content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -157,6 +157,16 @@ pointing to the Pod's IP address. Also, Pod needs to become ready in order to ha
 record unless `publishNotReadyAddresses=True` is set on the Service.
 {{< /note >}}
 
+### Pod's setHostnameAsFQDN field {#pod-sethostnameasfqdn-field}
+
+When a Pod is configured to have fully qualified domain name (FQDN), its hostname is the short hostname. For example, if you have a Pod with the fully qualified domain name `busybox-1.default-subdomain.my-namespace.svc.cluster-domain.example`, then by default the `hostname` command inside that Pod returns `busybox-1` and  the `hostname --fqdn` command returns the full FQDN.
+
+When you set `setHostnameAsFQDN: true` in the Pod spec, the kubelet writes the Pod's FQDN into the hostname for that Pod's namespace. In this case, both `hostname` and `hostname --fqdn` return the Pod's FQDN.
+
+{{< note >}}
+In Linux, the hostname field of the kernel (the nodename field of struct utsname) is limited to 64 characters. If a Pod enables this feature and its FQDN is longer than 64 character, it will fail to start. The Pod will remain in `Pending` status (`ContainerCreating` as seen by `kubectl`) generating error events regarding its FQDN length. One way of improving user experience for this scenario is to create an [admission webhook controller](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks) to control FQDN size when users create top level objects, for example, Deployment.
```

Comment:
nit (Ok to merge without making changes here)
```suggestion
In Linux, the hostname field of the kernel (the `nodename` field of `struct utsname`) is limited to 64 characters.

If a Pod enables this feature and its FQDN is longer than 64 character, it will fail to start. The Pod will remain in `Pending` status (`ContainerCreating` as seen by `kubectl`) generating error events regarding its FQDN length. One way of improving user experience for this scenario is to create an [admission webhook controller](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks) to control FQDN size when users create top level objects, for example, Deployment.
```

Reply:
thanks @sftim. I updated with the suggested change, but I think the `note` formatting does not allow breaks, nothing changed in the output.

Reply:
> I think the note formatting does not allow breaks, nothing changed in the output.

For context: The backticks were the main thing I wanted but I'm also hoping to fix the note shortcode soon so I suggested adding a newline too.

Reply:
(and I'll reiterate that it's OK to merge without addressing this very minor feedback).

Reply:
I missed the backticks, yeah it makes sense to add them. I just did. I will leave the new line as well.